Formula/htmltest.rb

```diff
@@ -16,7 +16,7 @@ class Htmltest < Formula
 
   def install
     ldflags = %W[
-      -X main.date=#{Utils.safe_popen_read("date", "-u", "+%Y-%m-%dT%H:%M:%SZ")}
+      -X main.date=#{Time.now.strftime("%Y-%m-%dT%H:%M:%S%Z")}
```

Comment:
I'm not sure these are the same (sorry, should have put this under the same review):

```shell
$ date -u '+%Y-%m-%dT%H:%M:%SZ'
2021-03-08T01:50:57Z

$ brew ruby -e 'puts Time.now.strftime("%Y-%m-%dT%H:%M:%S%Z")'
2021-03-07T20:51:13EST
```

Looks like a time-zone difference. How about this:

```suggestion
      -X main.date=#{Time.now.utc.strftime("%Y-%m-%dT%H:%M:%SZ")}
```

Reply:
Yup; I know they're not the same. I'm pretty sure the trailing `Z` is actually a typo, and it really needed to be `%Z`, for timezone.

Reply:
[ISO 8601](https://en.wikipedia.org/wiki/ISO_8601) does use that trailing `Z` to indicate UC time, so I think it was intentional to use the `Z` character over the `%Z` timezone.

Current time in ISO 8601 according to that page: `2021-03-08T02:06:27Z`

Reply:
Ah, yes, that's supposed to signify `UTC`. But it's technically a lie, though, since I think only our Intel nodes are on UTC...

Reply:
`date -u` uses UTC mode, regardless of the host timezone.

The `Z` matches what upstream does: https://github.com/wjdp/htmltest/blob/6297c3bf18d7afaf721a08ecdf750dd3a42757a1/build.sh#L5

Reply:
Ok, `utc` with the `Z` then.